docs/application-connector/docs/050-gs-calling-registered-service-from-kyma.md

```diff
@@ -0,0 +1,198 @@
+  deployment:
+    spec:
+      template:
+        metadata:
+          labels:
+            re-{RE_NAME}-{SERVICE_ID}: "true"
+        spec:
+          containers:
+          - env:
+            - name: GATEWAY_URL
+              value: re-{RE_NAME}-{SERVICE_ID}.kyma-integration
+  deps: |-
+    {
+        "name": "example-1",
+        "version": "0.0.1",
+        "dependencies": {
+          "request": "^2.85.0"
+        }
+    }
+  function: |-
+    const request = require('request');
+
+    module.exports = { main: function (event, context) {
+        return new Promise((resolve, reject) => {
+            const url = \`http://\${process.env.GATEWAY_URL}/uuid\`;
+            const options = {
+                url: url,
+            };
+              
+            sendReq(url, resolve, reject)
+        })
+    } }
+
+    function sendReq(url, resolve, reject) {
+        request.get(url, { json: true }, (error, response, body) => {
+            if(error){
+                resolve(error);
+            }
+            resolve(response.body);
+        })
+    }
+  function-content-type: text
+  handler: handler.main
+  horizontalPodAutoscaler:
+    spec:
+      maxReplicas: 0
+  runtime: nodejs8
+  service:
+    ports:
+    - name: http-function-port
+      port: 8080
+      protocol: TCP
+      targetPort: 8080
+    selector:
+      created-by: kubeless
+      function: my-lambda
+  timeout: ""
+  topic: http
+EOF
+```
+
+5. Create the Service Binding and the Service Binding Usage to bind the previously create Service Instance to the lambda.
+
+```
+cat <<EOF | kubectl apply -f -
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceBinding
+metadata:
+  labels:
+    Function: my-lambda
+  name: my-service-binding
+  namespace: production
+spec:
+  instanceRef:
+    name: my-service-instance-name
+EOF
+```
+
+```
+cat <<EOF | kubectl apply -f -
+apiVersion: servicecatalog.kyma.cx/v1alpha1
+kind: ServiceBindingUsage
+metadata:
+  labels:
+    Function: my-lambda
+    ServiceBinding: my-service-binding
+  name: my-service-binding
+  namespace: production
+spec:
+  serviceBindingRef:
+    name: my-service-binding
+  usedBy:
+    kind: function
+    name: my-lambda
+EOF
+```
+
+6. To expose the lambda outside the cluster create an Api resource:
+```
+cat <<EOF | kubectl apply -f -
+apiVersion: gateway.kyma.cx/v1alpha2
+kind: Api
+metadata:
+  labels:
+    function: my-lambda
+  name: my-lambda
+  namespace: production
+spec:
+  authentication: []
+  hostname: my-lambda-production.{CLUSTER_DOMAIN}
+  service:
+    name: my-lambda
+    port: 8080
+EOF
+```
+On Minikube as a cluster domain use `kyma.local`.
+
+7. To verify that everything was setup correctly you can now call the lambda through https:
+  - On cluster
+    ```
+    curl https://my-lambda-production.{CLUSTER_DOMAIN}/ -k
+    ```
+  - On Minikube
+    To be able to access the lambda over https edit `/etc/hosts/` file and map the `my-lambda-production.kyma.local` host to the Minikube ip.
```

Comment:
Move this to prerequisites. 

Map the `my-lambda-production.kyma.local` to your Minikube IP to call the lambda function on a local Kyma deployment.